lib/ansible/modules/crypto/luks_device.py

```diff
@@ -317,16 +334,21 @@ def is_luks(self, device):
 
     def run_luks_create(self, device, keyfile, keysize):
         # create a new luks container; use batch mode to auto confirm
-        label = self._module.params.get('label')
+        type = self._module.params['type']
```

Comment:
`type` is a reserved identifier in Python. You can use it, but it is not good style. Better use `luks_type` or something for the variable name. (Using `type` for the option name is totally fine.)

Reply:
Done